### PR TITLE
AArch64: Implement lneg Evaluator

### PR DESCRIPTION
**Pull request by marufunb**
Implemented lneg Evaluator under Unary Evaluator

Signed-off-by: marufunb <mrahma15@unb.ca>

### CODE REVIEW

**Comment by marufunb** [comment]
You can look at the tril test for lneg evaluator if needed at pull request #3321

File: compiler/aarch64/codegen/UnaryEvaluator.cpp [hunk 1]
```diff
@@ -77,8 +77,11 @@ TR::Register *OMR::ARM64::TreeEvaluator::inegEvaluator(TR::Node *node, TR::CodeG
 
 TR::Register *OMR::ARM64::TreeEvaluator::lnegEvaluator(TR::Node *node, TR::CodeGenerator *cg)
 	{
```
Comment (0xdaryl) [inline compiler/aarch64/codegen/UnaryEvaluator.cpp]: Can you fix the indenting of these braces to match the other functions please?

**Comment by 0xdaryl** [comment]
@genie-omr build aarch64

**Comment by 0xdaryl** [comment]
This is an AArch64-specific change.  A full genie build is not required.

**Pull request merged**